src/lifted/queue.py

```diff
@@ -31,11 +31,12 @@
 
 from operator import attrgetter
 import os
-import pickle
 import stat
 import time
 
-from lifted.upload import Upload, UploadStatus
+import toml
```

Comment:
We should use pylorax.api.toml which is our wrapper around the toml library that acts like pytoml did.